bluemira/base/reactor.py

```diff
@@ -101,7 +116,13 @@ def _build_component_tree(self) -> Component:
             component.add_child(component_manager.component())
         return component
 
-    def show_cad(self, *dims, **kwargs):
+    def show_cad(
+        self,
+        *dims,
+        with_components: Optional[List[ComponentManager]] = None,
+        with_n_sectors: Optional[int] = None,
```

Comment:
possibly just `n_sectors`? The with seems a bit redundant in at least that case maybe also for `with_components` but that one sits better with me.
```suggestion
        n_sectors: Optional[int] = None,
```